cmd/Makefile.am

```diff
@@ -1,3 +1,8 @@
 SUBDIRS  = zfs zpool zdb zhack zinject zstreamdump ztest
-SUBDIRS += mount_zfs fsck_zfs zvol_id vdev_id arcstat dbufstat zed
-SUBDIRS += arc_summary raidz_test zgenhostid
+SUBDIRS += fsck_zfs vdev_id raidz_test zgenhostid
+
+if USING_PYTHON
+SUBDIRS += arcstat arc_summary dbufstat
+endif
+
+SUBDIRS += mount_zfs zed zvol_id
```

Comment:
Why is this line separated from the others above?

Reply:
These are all the linux-specific subdirs, which soon can be wrapped in `if BUILD_LINUX`. This was an attempt at reducing the future diff while already touching these lines.

Reply:
Makes sense!